core/test/integration/index.ts

```diff
@@ -158,6 +158,52 @@ describe('Mongo Seeding', () => {
     });
   });
 
+  it('should remove all documents from collections before importing data', async () => {
```

Comment:
Currently the test doesn't actually test the `removeAllDocuments` option.
Please modify this test case:

- create at least 2 collections with at least 2 documents per each
- create at least 1 index per collection
- seed DB (e.g. with one document per collection) with `removeAllDocuments` option enabled
- see if:
   - all previous docs are removed & new documents are populated
   - indexes are kept